CONTRIBUTING.md

```diff
@@ -0,0 +1,31 @@
+# Contributing to r7insight_ruby
+
+:+1::tada: Thanks for taking the time to contribute! :tada::+1:
+
+## Requirements
+
+In order to work on this repository you will require:
+- ruby version `2.6.5`
+- bundle version `2.0.2`
+- gem version `3.0.6`
+- rvm version `1.29.9`
+
+## Workflow
+
+- Fork repository in GitHub
+- Clone your repository fork
+- Implement functionality
+- `make test` for testing
+- Add extra tests and documentation if required
+- Push into your fork and create a pull request into the main Rapid7 repository
+- Once pull request is approved `make bump-(major|minor|patch)` for bumping versions (use [SemVer](https://semver.org/))
```

Comment:
Only we can do this, so no point in publicly documenting this. Belongs on Confluence, I think.

Reply:
I'm guessing you're referring to the `Deployment information` below, true I'd agree.
But I think it's a lot better to have the information localised and available rather than needing to sieve through conf. Plus nothing confidential here.